src/game/behaviors/water_ring.inc.c

```diff
@@ -42,16 +43,21 @@ void bhv_jet_stream_water_ring_init(void) {
 }
 
 void water_ring_check_collection(UNUSED f32 avgScale, struct Object *ringManager) {
+#ifdef FIX_WATER_RINGS
+    if (o->oInteractStatus & INT_STATUS_INTERACTED) {
+#else
     f32 marioDistInFront = water_ring_calc_mario_dist();
 
     if (!is_point_close_to_object(o, gMarioObject->header.gfx.pos[0],
-                                  gMarioObject->header.gfx.pos[1] + 80.0f,
-                                  gMarioObject->header.gfx.pos[2], (avgScale + 0.2f) * 120.0f)) {
+                                     gMarioObject->header.gfx.pos[1] + 80.0f,
+                                     gMarioObject->header.gfx.pos[2],
```

Comment:
at some point we need the mario height defines to be used in these behavior files that use half height

Reply:
(non blocking obviously)